control-plane/roles/isolated-clusters/tasks/dns.yaml

```
@@ -0,0 +1,27 @@
+- name: Get kubeconfig
+  copy:
+    src: "files/kubeconfigs/kubeconfig.{{ cluster }}"
+    dest: "/tmp/kubeconfig.{{ cluster }}"
+
+- name: Create namespace {{ isolated_clusters_dns_namespace }}
+  k8s:
+    definition:
+      api_version: v1
+      kind: Namespace
+      metadata:
+        name: "{{ isolated_clusters_dns_namespace }}"
+        labels:
+          group: "{{ isolated_clusters_group_label }}"
+  environment:
+    K8S_AUTH_KUBECONFIG: "/tmp/kubeconfig.{{ cluster }}"
```

Please use key `kubeconfig`, because we pass the kubeconfig as a dictionary instead of a file:
```suggestion
- name: Create namespace {{ isolated_clusters_dns_namespace }}
  k8s:
    definition:
      api_version: v1
      kind: Namespace
      metadata:
        name: "{{ isolated_clusters_dns_namespace }}"
        labels:
          group: "{{ isolated_clusters_group_label }}"
    kubeconfig: "{{ kubeconfig as dictionary }}"
```